src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.js

```diff
@@ -757,7 +757,7 @@ describe('VAOS Page: CommunityCareProviderSelectionPage', () => {
 
     // Belgrade is the 2nd of three options so the expectation is
     // that it should be selected when we get to the CommunityCareProviderSelectionPage.
-    await setClosestCity(store, /Belgrade/i);
+    await setClosestCity(store, '984');
```

Comment:
You can remove the comment lines about Belgrade since 984 is Dayton

Reply:
I don't think the test use the real facility locations but the setup on line 753-754 specifies facility 984 as Belgrade. I think I'll update the comments to say "Facility 983/4" to more accurately reflect what it's referring to but overall, at least these city names won't affect the correctness of the test.